lib/onebox/normalizer.rb

```diff
@@ -4,19 +4,11 @@ module Onebox
   class Normalizer
     attr_reader :data
 
-    def get(attr, length = nil, sanitize = true)
-      return nil if Onebox::Helpers.blank?(data)
-
+    def get(attr, *args)
       value = data[attr]
-
-      return nil if Onebox::Helpers.blank?(value)
-
-      value = html_entities.decode(value)
-      value = Sanitize.fragment(value) if sanitize
-      value.strip!
-      value = Onebox::Helpers.truncate(value, length) unless length.nil?
-
-      value
+      return if value.blank?
+      return value.map { sanitize_value(_1, *args) } if Array === value
```

Comment:
While I love numbered parameters, I think they are slightly less easier to read than being explicit. What do you think?

```suggestion
      return value.map { |v| sanitize_value(v, *args) } if Array === value
```

Reply:
While I agree I wouldn’t use them all the time, I think this use case (small and simple one-liners) is perfect for their use. Do you think this is something that should be discussed as part of our coding lore?

Reply:
> Do you think this is something that should be discussed as part of our coding lore?

Excellent idea 👍 

Do you know if there's a rubocop rule for them? That way we could automate our decision (be it allow them, never allow them, or use them with caution)

Reply:
Indeed there’s one: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumberedParameters :slightly_smiling_face: The default seems to be a good one, it’s allowing them only for single line blocks.